bigquery/tests/system.py

```diff
@@ -1235,6 +1235,30 @@ def test_query_future(self):
         row_tuples = [r.values() for r in iterator]
         self.assertEqual(row_tuples, [(1,)])
 
+    def test_query_iter(self):
```

Comment:
This doesn't need to be a system test, it's just checking behavior of a magic method. (Unless you don't have an equivalent test that invoked `.result()`.)